packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -147,8 +147,8 @@ export class MultiSelect<T> extends React.Component<IMultiSelectProps<T>, IMulti
                     onKeyUp={this.state.isOpen ? handleKeyUp : undefined}
                 >
                     <TagInput
-                        inputProps={defaultInputProps}
                         {...tagInputProps}
+                        inputProps={defaultInputProps}
```

Comment:
moving this line does not feel correct as it prevents changing any of the fields defined in `defaultInputProps`. why did you do this?